WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -508,6 +510,17 @@ import WordPressComAnalytics
     }
 
 
+    // Refresh the header of a site topic when returning in case the
+    // topic's following status changed.
+    func refreshTableHeaderIfNeeded() {
+        if let siteTopic = readerTopic as? ReaderSiteTopic {
+            if let header = tableView.tableHeaderView as? ReaderStreamHeader {
```

Comment:
Nitpicky, but you could combine these into a single `if`:

```
if let siteTopic = readerTopic as? ReaderSiteTopic,
  let header = tableView.tableHeaderView as? ReaderStreamHeader {
  header.configureHeader(siteTopic)
}
```

Less indentation!